extension/src/popup/components/WarningMessages/styles.scss

```diff
@@ -156,6 +156,13 @@
     .BasicButton--primary {
       background: #e5484d;
       border-color: #e5484d;
```

Comment:
@piyalbasu  also looks like this got removed in https://github.com/stellar/freighter/pull/654/files 

but we need the hover state for the button otherwise turns blue. I went with `var(--pal-error);` hover color lmk if something else is better tho

Reply:
Ah yeah, you're right about the `hover`.

Why is 
 ```
 --Button-color-background: #e5484d;
      --Button-color-border: #e5484d;
 ``` 
 needed though?

Reply:
Ah you're right about `hover`

Let's use `--Button-color-background` and `--Button-color-border` since we're defining them